### ISSUE TITLE
Could not connect to development server. Using Android emulator. 

### ISSUE THREAD

**Issue by joshg111** [facebook/react-native#9089]
I just started a new project, with latest version of node.js, react, react-native, react-native-cli. My old projects were using some older versions of those packages and were working fine. 

react-native init <My Project Name>
cd <My Project Name>
react-native run-android

I see the app gets installed on the emulator. When the app launches on the emulator after some time it give the error "Could not connect to development server". 

My package.json:
```
{
  "name": "MyProjectName",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start"
  },
  "dependencies": {
    "react": "15.2.1",
    "react-native": "0.30.0"
  }
}
```

Output of "adb devices":
```
List of devices attached
emulator-5554   device
```

My Android sdk's are fairly new, but maybe I will try to reinstall all that stuff per react-native docs. 

Btw, the react-native docs don't specify how to start the emulator. I start the emulator by launching Android Studio, and opening the AVD manager. This approach seems to work with my old projects. 

Please let me know what other information is needed.

**Comment by joshg111** [comment]
Also, the packager server is running on port 8081.

**Comment by joshg111** [comment]
I think typically there is a request sent by the emulator when it is "Fetching JS Bundle" and that request is logged in stdout of the packager server, but now I do not see any request hitting the server.

**Comment by joshg111** [comment]
Turns out the TermService was using port 8081. Imo react native packager server should throw an error if another process is using the port it wants. 

Ticket can be closed.

**Comment by brookehurford** [comment]
@joshg111 Could you explain what you mean by TermService - or how you ended up fixing this? I am experiencing the same issue but have little experience on Android.

**Comment by joshg111** [comment]
I'm on windows so I ran some command to get used ports. 
`netstat -a -b > out`

Then I opened "out" and found that the TermService was using port 8081. TermService is used for windows remote desktop protocol. Turns out I coincidentally used port 8081 for remote desktop setup and didn't realize it would conflict with react native. I changed the port for remote desktop back to the default in regedit, and restarted, then everything worked fine.
